src/widgets/popOver.test.js

```diff
@@ -0,0 +1,248 @@
+import '../util/jsdomSetup';
+//import expect, {spyOn} from 'expect';
+import expect from 'expect';
+import def from './popOver.def.js';
+import lolex from 'lolex';
+// import equals from '../util/equals';
+import $ from 'jquery';
+
+describe('The popOver Widget', () => {
+	let pop, pow, clock, target, spy;
+	const testContainer = $('<div/>').appendTo(document.body);
+
+	const setup = (htmlString = '<div><p>Some content</p><br><h4>And some more content</h4></div>', options) => {
+		testContainer.empty();
+		target = $('<div id="target"><h1>This is a target!</h1></div>').appendTo(testContainer);
+
+		pop = $(htmlString);
+		pow = Object.create(def);
+
+		pow.element = pop;
+		pow.options = options || pow.options;
+		pow.options.target = target;
+
+		/****************************************************/
+		// Talk to charles about this needing to be done since trigger does not exist yet
+		pow._trigger = expect.createSpy();
+		/****************************************************/
+		
+		pow._create();
+		pow._init();
+
+		spy = {
+			trigger: expect.spyOn(pow, '_trigger').andReturn(true),
+			position: expect.spyOn(pow, 'position').andCallThrough(),
+			show: expect.spyOn(pow, 'show').andCallThrough(),
+			hide: expect.spyOn(pow, 'hide').andCallThrough(),
+			element: {
+				position: expect.spyOn(pow.element, 'position').andCallThrough(),
+				fadeIn: expect.spyOn(pow.element, 'fadeIn').andCallThrough()
+			}
+		};
+	};
+
+	beforeEach(() => {
+		clock = lolex.install();
+		setup();
+	});
+
+	afterEach(() => {
+		clock.uninstall();
+		pow._destroy();
+		pop.remove();
+	});
+
+	/****************************************************/
```

Comment:
Remove all section comments in this style and replace with single block comments `/* ... */`.  If the test doesn't pass make sure to skip it so we don't merge in failing tests.  Also add the phrase `TODO: ...` anywhere that needs us to come back to so we can easily find them by searching through the code base.